src/flyte/_trigger.py

```diff
@@ -2,10 +2,611 @@
 
 from dataclasses import dataclass
 from datetime import datetime
-from typing import Any, Dict, Mapping, Union
+from typing import Any, Dict, Literal, Mapping, Union
 
 import rich.repr
 
+Timezone = Literal[
```

Comment:
maybe naive questions: 
* why do we need to list all timezones here? 
* is this list fixed or can it change in future?
* if a user provides an invalid timezone(e.g. `cron = Cron("0 * * * *", timezone="blabla")`), will it fail at runtime?